evalai/utils/urls.py

```diff
@@ -0,0 +1,5 @@
+urls = {
```

Comment:
Any reason this is a dict and not an enum?
Two issues:
1. The `get_*` indicates verbs which should only be used for functions and methods as it makes reading code easier.
2. With an Enum you could do this:

```
from enum import Enum

class Urls(Enum):
    challenge_list = "/api/challenges/challenge/all"
    past_challenge_list = "/api/challenges/challenge/past"
    future_challenge_list = "/api/challenges/challenge/future"

print(Urls.challenge_list.value)
```

Looks pretty clean, IMHO.

Reply:
Using an enum never crossed my mind, I'll change it. :)